exec/java-exec/src/main/java/org/apache/drill/exec/planner/sql/parser/DrillCompoundIdentifier.java

```diff
@@ -69,31 +70,38 @@ public void addIndex(int index, SqlParserPos pos){
     }
   }
 
-  public SqlNode getAsSqlNode(){
-    if(ids.size() == 1){
+  public SqlNode getAsSqlNode(Set<DrillCompoundIdentifier> fullSchemasSet) {
```

Comment:
Can you add comment why you have to pass in the set of fullSchema? Without it, why will the previous code treat "cp" in "cp.`employee.json`" as a table name rather than schema name?